lib/src/data/article/article_cache_policy.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:allthenews/src/data/persistence/cache/cache_policy.dart';
+import 'package:allthenews/src/domain/model/article.dart';
+
+class ArticleCachePolicy extends CachePolicy<Article> {
+  @override
+  bool isValid(Article article) {
+    final currentDateTime = DateTime.now();
+    return article != null &&
+        article.updateDateTime.isAfter(
+          DateTime(
+            currentDateTime.year,
+            currentDateTime.month,
+            currentDateTime.day,
+            currentDateTime.hour,
```

Comment:
to zawsze zwróci false

Reply:
jeżeli current będzie 2020-09-28 15:57:00, a article będzie 2020-09-28 15:55:00 to isAfter zwróci true bo porówna z 2020-09-28 15:00:00 :)

Reply:
nie rozumiem, dlaczego tak porówna? w dokumentacji sekundy są brane pod uwagę https://api.dart.dev/stable/2.4.0/dart-core/DateTime/isAfter.html

Reply:
załóżmy:
final currentDateTime = "2020-09-28 15:56:56"
final article = "2020-09-28 15:52:10"
wtedy:
DateTime(
            currentDateTime.year,
            currentDateTime.month,
            currentDateTime.day,
            currentDateTime.hour,
) =  "2020-09-28 00:00:00"
czyli
"2020-09-28 15:52:10" isAfter "2020-09-28 00:00:00" zwróci true

Reply:
dobra już widzę, no będzie działać tak jak mówisz

Reply:
sprawdziłem sobie w dart padzie